tensorflow/python/keras/saving/hdf5_format.py

```diff
@@ -35,6 +35,7 @@
 from tensorflow.python.platform import tf_logging as logging
 from tensorflow.python.util import serialization
 from tensorflow.python.util.lazy_loader import LazyLoader
+import tensorflow.python.platform.gfile as gfile
```

Comment:
Let's do this as 
```suggestion
from tensorflow.python.platform.gfile import GFile
```
and then fix rest of code to work with this import.

This way, we follow the same import style as the other imports